wsmaster/che-core-api-account/src/test/java/org/eclipse/che/account/spi/AccountValidatorTest.java

```diff
@@ -62,39 +84,40 @@ public void testValidUserName(String input, boolean expected) throws Exception {
 
     @DataProvider
     public Object[][] namesToNormalize() {
-        return new Object[][] {{"test", "test"},
-                               {"test123", "test123"},
-                               {"test 123", "test123"},
-                               {"test@gmail.com", "testgmailcom"},
-                               {"TEST", "TEST"},
-                               {"test-", "test"},
-                               {"-test", "test"},
-                               {"te_st", "test"},
-                               {"te#st", "test"},
-                               {"-test", "test"},
-                               {"test-", "test"},
-                               {"--test--", "test"},
-                               {"t-----e--s-t", "t-e-s-t"}
+        return new Object[][]{{"test", "test"},
+                              {"test123", "test123"},
+                              {"test 123", "test123"},
+                              {"test@gmail.com", "testgmailcom"},
+                              {"TEST", "TEST"},
+                              {"test-", "test"},
+                              {"-test", "test"},
+                              {"te_st", "test"},
+                              {"te#st", "test"},
+                              {"-test", "test"},
+                              {"test-", "test"},
+                              {"--test--", "test"},
+                              {"t-----e--s-t", "t-e-s-t"},
+                              {"testTestTestTestTestTestTestTestTestTestTest", "testTestTestTestTestTestTestTestTestTes"}
```

Comment:
I'd prefer to use test value something like `f23456789012345678901234567890` then it'd be easier to understand this check.